.github/ISSUE_TEMPLATE/app_rendering_bug_report.yaml

```diff
@@ -0,0 +1,62 @@
+name: Application Card Rendering Bug Report
+description: Create a bug report for problems rendering Adaptive Cards in an Application
+title: "[Rendering] Bug Title Here"
+labels: [Area-Renderers, Bug, Triage-Needed]
+body:
+  - type: dropdown
+    validations:
+      required: true
+    attributes:
+      label: Target Platforms
+      description: Select the platform(s) that you are rendering on
+      multiple: true
+      options:
+        - NodeJS
+        - iOS
+        - Android
+        - WPF
+        - UWP
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: SDK Version
+      description: What version of the SDK are you using
+  - type: input
+    validations:
+      required: true
+    attributes:
+      label: Application Name
+      description: The name of the application that is having problems with Card Rendering
+      placeholder: Microsoft Teams
```

Comment:
is it possible to have a dropdown and still allow freeform input? if so, it would be nice if we could list each distinct Teams platform

Reply:
It doesn't appear that way, no...you either get freeform input OR a dropdown list of options, not both

Reply:
I added an option for `Other` as a fallback